templates/firefox-ios.xliff

```diff
@@ -2059,6 +2063,42 @@
         <source>Okay</source>
         <note>Restore Tabs Affirmative Action</note>
       </trans-unit>
+      <trans-unit id="Onboarding.IntroAction.v102" xml:space="preserve">
+        <source>Get Started</source>
+        <note>Describes the action on the first onboarding page in our Onboarding screen. This string will be on a button so user can continue the onboarding.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.IntroDescription.v102" xml:space="preserve">
+        <source>Indie. Non-profit. For good. Committed to the promise of a better Internet for everyone.</source>
+        <note>Description for the first onboarding page in our Onboarding screens.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.IntroDescriptionBold.v102" xml:space="preserve">
+        <source>Indie. Non-profit. For good.</source>
+        <note>Bold part of the description for the first onboarding page in our Onboarding screens. This string needs to be translated the same as Onboarding.introDescription where it duplicates it.</note>
```

Comment:
localizers don't necessarily look at the same strings. Please detail each time

Reply:
Passing by comment: I can't find any code using this string. Can you clarify how it's used? The whole "needs to be same" plus "bold" hints at text replacement, which would be a bad idea.

Reply:
@flodolo It is indeed text replacement. The requirement is that we have one string with a bold part in it. How would you suggest doing that?

Reply:
Can you have the markup in the string? 

If not, you should probably accept the compromise to have the string split in two parts, and ideally display them on separate lines (because concatenation is not always as safe).

```
<source>Indie. Non-profit. For good.</source>
<source>Committed to the promise of a better Internet for everyone.</source>
```

Also, look out for possible font issues with different scripts. I think bold is safer than italic, but I'm not sure if it's 100% safe (e.g. with Arabic).

Reply:
We splitted to two strings, let me know if it's ok!